gatsby/nice-icons.d.ts

```diff
@@ -0,0 +1,37 @@
+// nice-icons doesn't come with typescript defs
+// for the react components
+
+declare module "@nice-digital/icons/lib/Print" {
+	import { FC, FC, FC } from "react";
```

Comment:
Interesting! Is there a reason this needs to be imported 3 times or is it a mistake?

Reply:
That is absolutely a mistake! Will sort, good spot